peek/src/hooks/useRowSummaries.ts

```
@@ -161,136 +168,150 @@ export function useRowSummaries(columns: EsqlColumn[], visibleRows: unknown[][],
-      ...(currentProvider === "openrouter" ? { baseURL: "https://openrouter.ai/api/v1" } : {}),
-    });
-    const model =
-      currentProvider === "openrouter" ? openai.chat(currentModel) : openai(currentModel);
+        return !cacheRef.current.has(key);
+      });
+      if (pending.length === 0) return undefined;
 
-    for (const idx of batch) {
-      const row = visibleRows[idx];
-      if (!row) continue;
+      pending.forEach((idx) => inFlightIndicesRef.current.add(idx));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)